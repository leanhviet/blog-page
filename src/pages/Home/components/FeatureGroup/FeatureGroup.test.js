// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { FeatureGroupBase } from './index'

describe('FeatureGroupBase component', () => {
  const wrapper = shallow(<FeatureGroupBase />)

  it('should be defined FeatureGroupBase component', () => {
    expect(wrapper).toBeDefined()
  })

  // Testing Snapshot
  it('should be render snapshot FeatureGroupBase component', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
