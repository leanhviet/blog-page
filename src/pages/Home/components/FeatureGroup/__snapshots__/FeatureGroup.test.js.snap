// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureGroupBase component should be render snapshot FeatureGroupBase component 1`] = `
<div
  className="feature"
>
  <div
    className="feature__btn"
  >
    <Memo(DropdownBase)
      className="dropdown"
      onChange={[Function]}
      options={
        Array [
          Object {
            "name": "All",
            "value": "",
          },
          Object {
            "name": "Content",
            "value": "content",
          },
          Object {
            "name": "Title",
            "value": "title",
          },
          Object {
            "name": "Create At",
            "value": "createdAt",
          },
        ]
      }
    />
    <div>
      <Memo(ButtonBase)
        className="btn btn-primary"
        label="Desc"
        onClick={[Function]}
      />
      <Memo(ButtonBase)
        className="btn btn-primary"
        label="Asc"
        onClick={[Function]}
      />
    </div>
  </div>
  <div
    className="feature__search"
  >
    <Memo(InputBase)
      className="form-control"
      handleChange={[Function]}
      placeholder="Search..."
      type="text"
    />
  </div>
</div>
`;
